### PR TITLE
Updated recipe for version 1.8

### PR DESCRIPTION
**Pull request by jlstevens**
The PR will update the conda-forge version of HoloViews from 1.7 to 1.8. I've started with the changes to the recipe that will obviously be required.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,8 +7,7 @@ package:
 source:
   fn: holoviews-{{ version }}.tar.gz
   url: https://github.com/ioam/holoviews/archive/v{{version}}.tar.gz
-  sha256: c599716bc5ac7d9c342a3b8df35c2e9cbe2d349daef524d263e1ee4e33245ce8
-
+  sha256: a8bd74f0d099a39af059147953785be78662b0b3
```
Comment (ocefpaf) [inline recipe/meta.yaml]: This  does not looks like a `sha256` :confused:

**Comment by ocefpaf** [comment]
Please send PRs from a fork to avoid wasting CIs resources by triggering them twice and a premature upload of the package.

**Comment by jlstevens** [comment]
> Please send PRs from a fork to avoid wasting CIs resources by triggering them twice and a premature upload of the package.

Thanks for the heads up, I'll make sure to do that for the next release.

File: recipe/meta.yaml (hunk 1 above)
Comment (jlstevens) [inline recipe/meta.yaml]: Fixed. I mistakenly assumed shasum would use sha256 but default...

**Comment by jlstevens** [comment]
I've submitted the proposed recipe change from a fork in PR #8 as requested. Happy to see this PR closed.

**Comment by jlstevens** [comment]
The tests have failed so I am closing this PR in favor of #8.

**Pull request closed without merging**